CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## [1.5.0] - 2022-11-02
+
+### Added
+- X.509-SVID and JWT-SVID TTLs can now be configured separately at both the entry-level and Server default level (#3445)
+- `k8s-workload-registrar` and `oidc-discovery-provider` CLIs now print their version when the `-version` flag is set (#3475)
+- Support for customizing SPIFFE ID paths of SPIRE Agents attested with the `azure_msi` NodeAttestor plugin (#3488)
+
+###  Changed
+- External plugins can no longer be named `join_token` to avoid conflicts with the builtin plugin (#3469)
+- `spire-server run` command now supports DNS names for the configured bind address (#3421)
+- Documentation improvements (#3468, #3472, #3473, #3474, #3515)
+
+### Deprecated
+- `k8s-workload-registrar` is deprecated in favor of [SPIRE Controller Manager](https://github.com/spiffe/spire-controller-manager) (#3526)
+- Server `default_svid_ttl` configuration field is deprecated in favor of `default_x509_svid_ttl` and `default_jwt_svid_ttl` fields (#3445)
+- Entry `Ttl` field is deprecated in favor of `X509SvidTtl` and `JWTSvidTtl` fields (#3445)
```

Comment:
I think that this refers to the flags in the `spire-server create` command, right? I think that it would be good to explicit that, otherwise it's not clear where are those fields. Is this only a database change that doesn't affect me? etc.

Reply:
It's both for the Entry API type as well as the `spire-server create` and `spire-server update` CLI flags. I can update this to link to the actual field definitions in spire-api-sdk and use the snake-case style names rather than the Go CamelCase versions to try to clear that up. I was hesitating to call out all the CLI flags here since it's kind of implied that the CLIs should be able to support all the API fields.

Reply:
The `ttl` field has been removed (renamed) from the `Entry` message in the entry.proto definition, so it has been removed from the spire-api-sdk already. In SPIRE, the flag in the CLI commands is still there but deprecated. This is the kind of confusion that my comment was trying to point :)

Reply:
That's a good point. I think it's worth documenting both the API and CLI changes to clarify those nuances.